### ISSUE TITLE
Using @ then add comments faild.

### ISSUE THREAD

**Issue by fundon** [6to5/6to5#367]
```js
class A {

  say() {
    @walk();
    // do something, 
  }

  walk() {
  }

}
```

```console
TypeError: tests/yield.js: Cannot read property '1' of undefined
    at Controller.traverse.leave (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:800:35)
    at Controller.__execute (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:395:31)
    at Controller.traverse (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:483:28)
    at traverse (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:705:27)
    at Object.attachComments (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:794:9)
    at Object.exports.parse (/usr/local/lib/node_modules/6to5/lib/6to5/util.js:235:16)
    at File.parse (/usr/local/lib/node_modules/6to5/lib/6to5/file.js:178:15)
    at Object.transform (/usr/local/lib/node_modules/6to5/lib/6to5/transformation/transform.js:10:15)
    at Object.exports.transform (/usr/local/lib/node_modules/6to5/bin/6to5/util.js:23:20)
    at Object.exports.compile (/usr/local/lib/node_modules/6to5/bin/6to5/util.js:31:18)
```

**Comment by sebmck** [comment]
Seems to be an estraverse bug, I'll investigate shortly.

**Comment by sebmck** [comment]
Thanks! Fixed as of 2.4.7.
